src/policy/rbf.cpp

```diff
@@ -57,17 +57,18 @@ std::optional<std::string> GetEntriesForConflicts(const CTransaction& tx,
     uint64_t nConflictingCount = 0;
```

Comment:
a few lines up: Could also fixup the whitespace after the refactors in 2/3? https://github.com/bitcoin/bitcoin/pull/22675#pullrequestreview-747002094

Reply:
Oh right! Done, sorry I missed that